docs/fundamentals/tools-and-productivity.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tools and productivity in .NET
+author: IEvangelist
+description: Learn about the various tooling, software development kits, diagnostic tools, and code analyzers available to .NET developers.
```

Comment:
Tooling and diagnostic tools seems repetitive.

Reply:
```suggestion
description: Learn about the development and diagnostic tools available to .NET developers.
```